File: Modul-3/Python/PRAK304-2310817320007-ZahraNabila.py
x = int(input("Masukkan Bilangan : "))

if x == 0:
    print("Termasuk Bilangan : Nol")
    
elif 0 < x <= 9:
    print("Termasuk Bilangan : Satuan")

elif 10 < x < 20:
    print("Termasuk Bilangan : Belasan")

elif x == 10:
    print("Termasuk Bilangan : Puluhan")

elif 20 <= x < 100:
    print("Termasuk Bilangan : Puluhan")

elif x >= 100:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Modul-3/C/PRAK304-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    int x;
    printf("Masukkan Bilangan : ");
    scanf("%d", &x);

    {
        if ( x == 0)
        {
            printf("Termasuk Bilangan : Nol");
        }
        else if ( x >= 1 && x <= 9)
        {
            printf("Termasuk Bilangan : Satuan");
        } 
        else if (x > 10 && x < 20) 
        {
            printf("Termasuk Bilangan : Belasan");
        } 
        else if ( x == 10 )
        {
            printf("Termasuk Bilangan : Puluhan");
        } 
        else if ( x >= 20 && x < 100 )
        {
            printf("Termasuk Bilangan : Puluhan");
        } 
        else if ( x >= 100 )
        {
            printf("Anda Menginput Melebihi Limit Bilangan");
        } 
    }
}       